test/DGmethods/Euler/isentropic_vortex_standalone.jl

```diff
@@ -175,7 +175,13 @@ function main(mpicomm, DFloat, topl::AbstractTopology{dim}, N, timeend,
     outprefix = @sprintf("vtk/isentropicvortex_%dD_mpirank%04d_step%04d",
                          dim, MPI.Comm_rank(mpicomm), step[1])
     @debug "doing VTK output" outprefix
-    writevtk(outprefix, Q, spacedisc, statenames)
+    DGBalanceLawDiscretizations.writevtk(outprefix, Q, spacedisc, statenames)
```

Comment:
Can we remove `DGBalanceLawDiscretizations.`?

Reply:
yes, that should be removed